### ISSUE TITLE
createViewModel: remove restored property from localValues

### ISSUE THREAD

**Issue by apsavin** [mobxjs/mobx-utils#173]
```js
const model = { property: true };
const viewModel = createViewModel(model);
viewModel.property = false;
viewModel.property = true;
```

Actual result: 
```js 
viewModel.isDirty === true
```
Expected result: 
```js 
viewModel.isDirty === false
```

It seems like an easy to fix issue, I can come with a PR if you'll accept it.

**Comment by ItamarShDev** [comment]
closing as fixed was merged

**Comment by mweststrate** [comment]
Released as 5.2.0
